tools/sgen/input/FluentUIStyle.yml

```diff
@@ -21,6 +21,9 @@ Colors:
     - brandPressed: "FluentUIColor(light: $Colors.Brand.shade30, dark: $Colors.Brand.shade30)"
     - brandSelected: "FluentUIColor(light: $Colors.Brand.shade20, dark: $Colors.Brand.shade20)"
     - brandDisabled: "FluentUIColor(light: NamedColor(grey88), dark: NamedColor(grey84))"
+    - surfacePrimary: "FluentUIColor(light: NamedColor(white), dark: NamedColor(black), darkElevated: NamedColor(gray950))"
```

Comment:
Aren't we supposed to use the Vnext neutral palette? (grey2 to grey98) instead of gray950?

Reply:
Vnext neutral palette is only used for left nave (teams scenarios). We still default to primary colors for popover background and navigation background color.

Reply:
I have updated the colors from new catalog